### ISSUE TITLE
Newline

### ISSUE THREAD

**Issue by Cool-Game-Dev** [marty-oehme/bemoji#41]
Whenever I copy the emoji using rofi, it also copies a newline with it. Is there a fix?

**Comment by marty-oehme** [comment]
Hi @Cool-Game-Dev, do you mean when you invoke bemoji with the `-c` option to copy the emoji to the clipboard (`bemoji -n -c`) or when you use the rofi shortcut `Alt+1` to copy it into the clipboard?

If it is the former, please remember to supply the `-n` option to remove trailing newlines.

However, I assume you mean the latter, and I think you did indeed spot a bug in the code. There are two issues here: the echo options are not correctly applied (Line [309](https://github.com/marty-oehme/bemoji/blob/main/bemoji#L309) and the following 312) for the respective commands, and currently the echo options themselves are only set for the main command instead of for each (Line [304](https://github.com/marty-oehme/bemoji/blob/main/bemoji#L304)). These issues are definitely fixable and I can get around to them when I have some more time again.

**Comment by Cool-Game-Dev** [comment]
@marty-oehme While I was not using Alt+1, I was pressing enter, which seems to do the same thing.

**Comment by marty-oehme** [comment]
Yes, the default 'enter' action is sending to clipboard. The `-n` flag is the important one which removes the trailing newline, as described [here](https://github.com/marty-oehme/bemoji?tab=readme-ov-file#do-not-skip-to-new-line-after-output).
